package.json

```diff
@@ -38,7 +38,7 @@
     "tsc:build": "yarn pnpify tsc --build",
     "clean": "yarn pnpify tsc --build --clean",
     "lint": "yarn pnpify eslint src --ext .ts",
-    "start": "yarn build && yarn live-server --host=localhost --entry-file=./index.html",
+    "start": "yarn build && concurrently 'WATCH=true yarn build' 'yarn live-server --host=localhost --entry-file=./index.html'",
```

Comment:
`yarn dev`?

Reply:
dont think it really matters here, theres no server, yarn start is more standard than yarn dev imo